### DISCUSSION TITLE
Track Swap Widget

### DISCUSSION THREAD

**Discussion by roomrys** [talmolab/sleap#1558] [Ideas]
There have been a few discussions flying around about duplicate tracks when merging (tracked in #1083). What if we had a widget that allowed users to mass swap tracks of instances? Something like we have for swapping a skeleton for a template skeleton? Of course, this might be a mute feature if we just handle #1083 upon merging.

**Comment by gitttt-1234** [comment]
Closing this one! The root problem (duplicate tracks on merge, #1083) has been fixed in sleap-io. `Labels.merge()` now defaults to `TrackMatcher(method="name")`, which automatically deduplicates tracks by name during merge. See [`Merging guide`](https://io.sleap.ai/latest/merging/#track-matching) for details.
